aclgen.py

```diff
@@ -46,10 +46,14 @@
 from lib import packetfilter
 from lib import pcap
 from lib import policy
+from lib import policyparser
 from lib import speedway
 from lib import srxlo
 from lib import windows_advfirewall
 
```

Comment:
This policyparser is how Capirca .pol policies actually get parsed.  If I remove this.